tests/suits/convert/getQuote.test.ts

```diff
@@ -0,0 +1,243 @@
+import { expect } from "chai";
+import { ethers } from "ethers";
+import { Network } from "../../../src";
+import Convert from "../../../src/components/Convert";
+import { PerpInfoMethods } from "../../../src/components/Trade/types";
+import { PERP_TOKENS, PRICE_DECIMALS } from "../../../src/perp-config";
+import { getHlpToken, getNativeWrappedToken } from "../../../src/utils/perp";
+import { getTokenDetails } from "../../../src/utils/token-utils";
+
+const convert = new Convert();
+const weth = getNativeWrappedToken("arbitrum")!; // arbitrum
+const eth = PERP_TOKENS["arbitrum"].find((x) => x.isNative)!;
+const hlp = getHlpToken("arbitrum");
+const fxUsd = PERP_TOKENS["arbitrum"].find((x) => x.symbol === "fxUSD")!;
+const fxAud = PERP_TOKENS["arbitrum"].find((x) => x.symbol === "fxAUD")!;
+
+const FIVE_DOLLARS = ethers.utils.parseUnits("5", PRICE_DECIMALS);
+const ONE_DOLLAR = ethers.utils.parseUnits("1", PRICE_DECIMALS);
+
+const samplePerpTokenMethods: PerpInfoMethods = {
+  getMinPrice: () => ONE_DOLLAR,
+  getMaxPrice: () => ONE_DOLLAR,
+  getAveragePrice: () => ONE_DOLLAR,
+  getFundingRate: () => ethers.constants.One,
+  getTokenInfo: () => undefined,
+  getUsdgSupply: () => ethers.constants.One,
+  getTargetUsdgAmount: () => ethers.constants.One,
+  getTotalTokenWeights: () => ethers.constants.One,
+  getHlpPrice: () => FIVE_DOLLARS
+};
+
+describe("convert getQuote", () => {
+  describe("WETH", () => {
+    it("should be 1-1 from eth to weth", async () => {
+      const { quote, feeBasisPoints } = await convert.getQuote(
+        {
+          fromToken: weth,
+          toToken: eth,
+          canUseHlp: false,
+          network: "arbitrum",
+          connectedAccount: "0x0000000000000000000000000000000000000000",
```

Comment:
```suggestion
          connectedAccount: ethers.constants.AddressZero,
```
Please refactor all of these to the ethers constant

Reply:
done